frontstage/views/contact_us.py

```diff
@@ -22,7 +26,16 @@
 
 @contact_us_bp.route("/", methods=["GET"])
 def contact_us():
-    return render_template("contact-us.html")
+    authorization = False
+    if session_key := request.cookies.get("authorization"):
+        redis_session = Session.from_session_key(session_key)
+        try:
+            validate_jwt(redis_session, session_key)
+            authorization = True
+        except (Unauthorized, DecodeError, JWTValidationError, JWTTimeoutError):
+            pass
```

Comment:
Thought about putting logging in here, but I think it is pointless considering the context. This is to either to display an email or a hyperlink to the secure message form if authorised. I can see not scenario when we debug this, If it was a decorated endpoint maybe, but realistically we have extended a non @jwt_authorization route to decide if a hyperlink is shown.